data/data/agent/systemd/units/set-hostname.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Agent-based installer hostname update service
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+ExecStart=/usr/local/bin/set-hostname.sh
+ExecStartPost=/bin/sleep 5
```

Comment:
Why this sleep?

Reply:
It is not necessary. Will remove. Thanks.